nvfuser_python_utils/__init__.py

```diff
@@ -1,3 +0,0 @@
-from .patch_nvfuser import patch_installation
```

Comment:
nitpick. Instead of just blindly removing this, maybe we can add a no-op here that throws a deprecation message?

Reply:
I'm not holding a strong opinion here. I'm just wondering how people can check whether we have a patch-nvfuser module or not.

Can we check nvfuser version before patch/import on before the changes here?